python/lsst/analysis/tools/actions/vector/calcRhoStatistics.py

```diff
@@ -21,17 +21,15 @@
 
 from __future__ import annotations
 
-__all__ = (
-    "CalcRhoStatistics",
-    "TreecorrConfig",
```

Comment:
Because we committed to `TreecorrConfig` being available from here, trying to import `TreecorrConfig` from here should still continue to work (with a deprecation warning) until the next major release.